arelle/ErrorManager.py

```diff
@@ -204,13 +204,22 @@ def propValues(properties: Any) -> Any:
                             _arg:ModelObject = arg.modelObject if isinstance(arg, ObjectPropertyViewWrapper) else arg
                             if len(modelObjectArgs) > 1 and getattr(arg,"tag",None) == "instance":
                                 continue # skip IXDS top level element
-                            fragmentIdentifier = cast(str, XmlUtil.elementFragmentIdentifier(_arg))
-                            if not hasattr(_arg, 'modelDocument') and _arg.namespaceURI == XbrlConst.svg:
+                            fragmentIdentifier = "#" + cast(str, XmlUtil.elementFragmentIdentifier(_arg))
+                            if not hasattr(_arg, 'modelDocument') and _arg.namespaceURI == XbrlConst.svg and len(refs) > 0:
                                 # This is an embedded SVG document without its own file.
-                                ref["href"] = "#" + fragmentIdentifier
+                                # Set the href to the containing document element that defined the encoded SVG.
+                                # and define a nestedHrefs attribute with the fragment identifier.
+                                priorRef = refs[-1]
+                                ref["href"] = priorRef["href"]
+                                priorNestedHrefs = priorRef.get("customAttributes", {}).get("nestedHrefs", [])
+                                ref["customAttributes"] = {
+                                    "nestedHrefs": [*priorNestedHrefs, fragmentIdentifier]
```

Comment:
Out of curiosity, where does the odd `['#element(/1/1)']` string format come from?  It seems like it would be nice if it were an `xs:list` `xs:anyURI`.

I don't completely follow why `XbrlConst.svg` is treated specially by the error handling code, but I guess it's the only case of an embedded doc with nested content that we have?

Reply:
> Out of curiosity, where does the odd ['#element(/1/1)'] string format come from? It seems like it would be nice if it were an xs:list xs:anyURI.

I think it’s based on the [XPointer element() scheme](https://www.w3.org/TR/xptr-element/), but Herm would know for sure. 

> I don't completely follow why XbrlConst.svg is treated specially by the error handling code, but I guess it's the only case of an embedded doc with nested content that we have?

That’s correct. I’m not aware of any other embedded object types. There’s already some handling elsewhere in the logger for objects with missing model documents, but it’s unclear when or why that code is triggered. I didn’t have time to dig into it, and I didn’t want to change behavior for those objects without a full understanding.